### PR TITLE
update README with PR info

### PR DESCRIPTION
**Pull request by NoelMacwan**

### CODE REVIEW

File: README.mkdn [hunk 1]
```diff
@@ -1,50 +1,25 @@
-CyanogenMod
+webOS
```
Comment (morphis) [inline README.mkdn:1]: webOS -> webOS ports

File: README.mkdn [hunk 2]
```diff
@@ -1,50 +1,25 @@
...
-For more help on using this tool, use this command: repo help upload
-
-Make your changes and commit with a detailed message, starting with what you are working with (i.e. vision: Update Kernel)
-Commit your patches in a single commit. Squash multiple commit using this command: git rebase -i HEAD~<# of commits>
-
-To view the status of your and others' patches, visit [CyanogenMod Code Review](http://review.cyanogenmod.org/)
-
+Pull Requests
+---------------
+We are here working to make this ROM as much good as we can and we would love to see you here to contribute.
```
Comment (morphis) [inline README.mkdn:6]: We don't call us a ROM. Just say: We try to make webOS ports as good as we can and we would love to you to contribute.

File: README.mkdn [hunk 3]
```diff
@@ -1,50 +1,25 @@
...
+So your commmits are always welcome !
+But before sending pull request, you are supposed to test it first 
+To get faimiliar with [creating pull request](https://help.github.com/articles/creating-a-pull-request)
 
 Getting Started
 ---------------
 
-To get started with Android/CyanogenMod, you'll need to get
+To get started with Android/webOS, you'll need to get
```
Comment (morphis) [inline README.mkdn:14]: webOS -> webOS ports

File: README.mkdn [hunk 4]
```diff
@@ -1,50 +1,25 @@
...
 familiar with [Git and Repo](http://source.android.com/source/using-repo.html).
 
-To initialize your local repository using the CyanogenMod trees, use a command like this:
+To initialize your local repository using the webOS trees, use a command like this:
```
Comment (morphis) [inline README.mkdn:17]: webOS -> webOS ports

File: README.mkdn [hunk 5]
```diff
@@ -1,50 +1,25 @@
...
-For more information on this Github Organization and how it is structured, 
-please [read the wiki article](http://wiki.cyanogenmod.org/w/Github_Organization)
-
-Buildbot
---------
-
-All supported devices are built nightly and periodically as changes are committed to ensure the source trees remain buildable.
-
-You can view the current build statuses at [CyanogenMod Jenkins](http://jenkins.cyanogenmod.org/)
+Please see the [webOS Wiki](http://webos-ports.org/wiki/Porting_Guide) for building instructions.
```
Comment (morphis) [inline README.mkdn:25]: webOS -> webOS ports

**Comment by morphis** [comment]
Comments apply for the other PR too.

**Comment by morphis** [comment]
Otherwise looks good.

**Comment by NoelMacwan** [comment]
so do i send new ?

**Comment by morphis** [comment]
You can simply append here.

**Comment by Herrie82** [comment]
Outdated

**Pull request closed without merging**